internal/cli/atlas/setup/setup_cmd.go

```diff
@@ -88,16 +94,18 @@ func Builder() *cobra.Command {
 	cmd.Flags().BoolVar(&opts.login.NoBrowser, "noBrowser", false, "Don't try to open a browser session.")
 	cmd.Flags().BoolVar(&opts.login.SkipConfig, "skipConfig", false, "Skip profile configuration.")
 	// Quickstart related
-	cmd.Flags().StringVar(&opts.quickstart.ClusterName, flag.ClusterName, "", usage.ClusterName)
-	cmd.Flags().StringVar(&opts.quickstart.Tier, flag.Tier, quickstart.DefaultAtlasTier, usage.Tier)
-	cmd.Flags().StringVar(&opts.quickstart.Provider, flag.Provider, "", usage.Provider)
-	cmd.Flags().StringVarP(&opts.quickstart.Region, flag.Region, flag.RegionShort, "", usage.Region)
-	cmd.Flags().StringSliceVar(&opts.quickstart.IPAddresses, flag.AccessListIP, []string{}, usage.NetworkAccessListIPEntry)
-	cmd.Flags().StringVar(&opts.quickstart.DBUsername, flag.Username, "", usage.DBUsername)
-	cmd.Flags().StringVar(&opts.quickstart.DBUserPassword, flag.Password, "", usage.Password)
-	cmd.Flags().BoolVar(&opts.quickstart.SkipSampleData, flag.SkipSampleData, false, usage.SkipSampleData)
-	cmd.Flags().BoolVar(&opts.quickstart.SkipMongosh, flag.SkipMongosh, false, usage.SkipMongosh)
-	cmd.Flags().BoolVar(&opts.quickstart.Confirm, flag.Force, false, usage.Force)
+	qsOpts := &quickstart.Opts{}
```

Comment:
[q] cuold we move this to the beginning of the command to have everything initialized at the same place? 
```golang
qsOpts := &quickstart.Opts{}
loginOpts := &auth.LoginOpts{}
opts := &Opts{
		register: auth.NewRegisterFlow(loginOpts),
		login:    loginOpts,
                quickstart: qsOpts
	}
```